web/page0/src/app/runtime.service.ts

```diff
@@ -188,28 +194,79 @@ export class RuntimeService {
     this.refresh();
   }
 
+  private getGroups(hostUrl: string) {
+    return this.apiService
+      .listGroups(hostUrl)
+      .pipe(map(({ groups }) => groups));
+  }
+
+  private getHosts(
+    runtimeUrl: string,
+    zone: string,
+    runtimeAlias: string
+  ): Observable<Host[]> {
+    return this.apiService.listHosts(runtimeUrl, zone).pipe(
+      mergeMap(({ items: hosts }) => {
+        return forkJoin(
```

Comment:
The statement `() => return forkJoin(....)` can be replaced with `() => forkJoin(....)`

Reply:
Parentheses are left to add console.log in handy.
Will apply on lint options all at once if available.

Reply:
Acknowledged. Thank you.